portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ListObjects.tsx

```diff
@@ -640,6 +651,12 @@ const ListObjects = ({
       disableButtonFunction: (item: string) =>
         extensionPreview(item) === "none",
     },
+    {
+      type: "share",
+      onClick: openShare,
+      disableButtonFunction: (item: string) =>
+        extensionPreview(item) === "none",
+    },
```

Comment:
We can remove this and apply the disable only to folders:

<img width="1446" alt="Screen Shot 2021-09-15 at 21 42 58" src="https://user-images.githubusercontent.com/33497058/133540921-8f16925b-2ad6-4a6b-828a-eaae53f8a63b.png">

```suggestion
      disableButtonFunction: (item: string) =>
        item.endsWith("/")
    },
```